defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 # file: postgresql/defaults/main.yml
 
 # Basic settings
-postgresql_version: 9.3
+postgresql_version: 9.6            # if you use postgis too check whether the postgresql/postgis versions work together
```

Comment:
Probably don't need that postgis comment in there.  There is a PR #310 , which will give reasonable defaults for PostGIS now.